CHANGES_NEXT_RELEASE

```diff
@@ -2,3 +2,4 @@
 - Hardening: Mongo driver now compiled using --use-sasl-client --ssl to enable proper DB authentication mechanisms
 - Fix: correct error payload using errorCode (previously orionError was used) in POST /v1/queryContext and POST /v1/updateContext in some cases
 - Hardening: modification of the URL parsing mechanism, making it more efficient, and the source code easier to follow (#3109, step 1)
+- Fix : Fix for Missing or empty metadata values are not allowed with POST /v2/op/update (#3121)
```

Comment:
I'd suggest the following wording, which I think is more precise:

```
- Fix : Missing or empty metadata values were not allowed in NGSIv1 create/update operations (#3121)
```

Reply:
fixed in 991cd65